spec/text_interface.go

```diff
@@ -13,6 +13,7 @@ type TextRequest interface {
 	// GetInput returns the input of the current text request.
 	GetInput() string
 
+	// TODO remove
```

Comment:
We can remove the json interfaces, because they are not needed. `gRPC` is taking care of message marshalling/unmarshalling.